client/src/components/Header.js

```diff
@@ -13,6 +16,67 @@ const Header = () => {
       </nav>
     </div>
   );
+
 };
 
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      auth: '',
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('auth/current_user')
+      .then(user => {
+        this.setState({ auth: user.data });
+        console.log(user);
```

Comment:
remove this log, you dont want the user to his data on the console :sweat_smile:

Reply:
Done